server/content-api/services/by-path.ts

```diff
@@ -34,6 +34,7 @@ export default () => ({
           $notNull: true,
         } : {},
       },
+      locale: 'all',
```

Comment:
Maybe this has something to do with something else, but why is this necessary? Isn't Strapi always using all locales?

Reply:
Oh nvm, I just checked, it is not by default sorry.

Reply:
Don't worry. I'ts a good point of discussion.
I made the assumption that it is best to query all pages of all languages in the `/all` endpoint.
Do you agree?

Reply:
I defenitely agree and I also agree that it makes sense to query all locales when looking for a specific path :)